doc/api/globals.md

```diff
@@ -1168,8 +1168,8 @@ added: v22.4.0
 
 > Stability: 1.0 - Early development.
 
-A browser-compatible implementation of [`Storage`][]. Enable this API with the
-[`--experimental-webstorage`][] CLI flag.
+A browser-compatible implementation of [`Storage`][]. Disable this API with the
```

Comment:
I think we need a test to ensure that when the flag `--no-experimental-webstorage` is provided, the API is, indeed, disabled.